src/main/scala/utils/images/Image.scala

```diff
@@ -282,7 +282,7 @@ trait VectorizedImage extends Image {
 
   def vectorToImageCoords(v: Int): Coordinate
```

Comment:
Can we make this method and the `coord` object `private` ? Right now `coord` is only used by this method, so it gives us some assurance it won't be accessed outside of it.